Piwik.Tracker/PiwikTracker.cs

```diff
@@ -1280,6 +1279,7 @@ protected bool LoadVisitorIdCookie()
             {
                 _visitCount = long.Parse(parts[2]);
             }
+            // todo _currentVisitTs is never used (was earlier assigned with null in constructor)

```

Comment:
I agree to letting it as it is, but documenting the fact that this field is just for information purposes.